llap-server/src/java/org/apache/hadoop/hive/llap/io/encoded/OrcEncodedDataReader.java

```diff
@@ -516,21 +471,13 @@ private static Object determineFileId(Supplier<FileSystem> fsSupplier, FileSplit
    * Closes the stripe readers (on error).
    */
   private void cleanupReaders() {
-    if (stripeReader != null) {
-      try {
-        stripeReader.close();
-      } catch (IOException ex) {
-        // Ignore.
-      }
-    }
-    if (rawDataReader != null && isRawDataReaderOpen) {
-      try {
-        rawDataReader.close();
-        rawDataReader = null;
-      } catch (IOException ex) {
-        // Ignore.
-      }
+    try {
+      IOUtils.close(stripeReader, rawDataReader, orcReader);
+    } catch (IOException e) {
+      LOG.warn("Error while closing readers: ", e);
     }
+    rawDataReader = null;
```

Comment:
what about `stripeReader` should we set it to null as well

Reply:
Done!